unit/models/application.py

```diff
@@ -12,18 +12,21 @@
 
 ApplicationTypes = Literal["individualApplication", "businessApplication"]
 
+
 class IndividualApplicationDTO(object):
     def __init__(self, id: str, created_at: datetime, full_name: FullName, address: Address, date_of_birth: date,
                  email: str, phone: Phone, status: ApplicationStatus, ssn: Optional[str], message: Optional[str],
                  ip: Optional[str], ein: Optional[str], dba: Optional[str],
                  sole_proprietorship: Optional[bool], tags: Optional[Dict[str, str]],
-                 relationships: Optional[Dict[str, Relationship]]):
+                 relationships: Optional[Dict[str, Relationship]], archived: Optional[bool],
+                 power_of_attorney_agent: Optional[Agent], id_theft_score: Optional[int]):
         self.id = id
         self.type = "individualApplication"
         self.attributes = {"createdAt": created_at, "fullName": full_name, "address": address,
                            "dateOfBirth": date_of_birth, "email": email, "phone": phone, "status": status, "ssn": ssn,
-                           "message": message, "ip": ip, "ein": ein, "dba": dba,
-                           "soleProprietorship": sole_proprietorship, "tags": tags}
+                           "message": message, "ip": ip, "ein": ein, "dba": dba, "archived": archived,
+                           "powerOfAttorneyAgent": power_of_attorney_agent, "soleProprietorship": sole_proprietorship,
+                           "idTheftScore": id_theft_score, "tags": tags}
```

Comment:
Hey @axshani - are we missing `nationality` here?

Reply:
You are right, one commit was missing
thank you for the notice